go.mod

```diff
@@ -52,8 +52,7 @@ require (
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/ini.v1 v1.62.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
-	gotest.tools v2.2.0+incompatible
```

Comment:
Looks like this was accidentally introduced in https://github.com/docker/compose-cli/commit/8961805412ba307255f378680a1af903b19b384c ( https://github.com/docker/compose-cli/pull/689)